src/Dev/DataExtensionAddsTrigger.php

```diff
@@ -0,0 +1,27 @@
+<?php
+
+namespace SilverStripe\StaticPublishQueue\Dev;
+
+use SilverStripe\Dev\TestOnly;
+use SilverStripe\ORM\DataExtension;
+use SilverStripe\StaticPublishQueue\Contract\StaticPublishingTrigger;
+use SilverStripe\StaticPublishQueue\Contract\StaticallyPublishable;
+
+class DataExtensionAddsTrigger extends DataExtension implements StaticallyPublishable, StaticPublishingTrigger, TestOnly
```

Comment:
Why do we add a new extension? The issue implies that the intention is for developers to be able to make their _own_ extension classes. Why do we need one in the module itself?

Reply:
This implements the TestOnly interface. It was only for the added test.
Let me know if there is a better way to test with Extensions.